homeassistant/components/switch/snmp.py

```diff
@@ -36,6 +39,18 @@
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_BASEOID): cv.string,
+    vol.Optional(CONF_COMMAND_OID, default=None):
+        vol.Any(
+            cv.string,
+            None),
+    vol.Optional(CONF_COMMAND_PAYLOAD_ON, default=None):
+        vol.Any(
+            cv.string,
+            None),
+    vol.Optional(CONF_COMMAND_PAYLOAD_OFF, default=None):
+        vol.Any(
+            cv.string,
+            None),
```

Comment:
Don't set it default None and do not allow set None as payload like others. Please use the same style